e2etests/pkg/address/address.go

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier:Apache-2.0
```

Comment:
in general the pattern seems to `E2E:`

Reply:
what pattern?

Reply:
sorry, I have been missing  the comments that I need to reword before submit the code review.
I am talking about the commit message it has `E2E Tests: test`, the pattern in previous commits is only `E2E:`

Reply:
this